plotter/invertedaxis_example_test.go

```diff
@@ -0,0 +1,57 @@
+// Copyright ©2017 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package plotter_test
+
+import (
+	"image/color"
+	"log"
+	"math"
+	"testing"
+
+	"gonum.org/v1/plot"
+	"gonum.org/v1/plot/internal/cmpimg"
+	"gonum.org/v1/plot/plotter"
+	"gonum.org/v1/plot/vg"
+)
+
+// Example_invertedScale shows how to create a plot with an inverted Y-axis.
+// This is nearly identical to the Log example, except it inverts the Y axis.
```

Comment:
I think the second sentence should be deleted, and the first sentence changed to say that both axes are inverted.

Reply:
👍